### ISSUE TITLE
Submitting emojis causes email notifications to loop

### ISSUE THREAD

**Issue by rickmerkelbach** [verbb/formie#1045]
### Describe the bug

Whenever a user submits an emoji in a (single or multiline) text field while "Sent Notifications" is enabled, the "Sending form notification" task does not get removed from the queue. It runs into an error when saving the "Sent Notification" entry. Whenever a worker daemon or cron job processes the queue, it sends the e-mail again and again.

When disabling the notification queue, Formie logs this error:
`2022-08-10 17:42:41 [-][1][-][error][formie] Notification email “Test” could not be sent for submission “149994”. Error: SQLSTATE[22007]: Invalid datetime format: 1366 Incorrect string value: '\xF0\x9F\xA4\xAF\x0A\x0A...' for column 'db'.'formie_sentnotifications'.'body' at row 1`

When using a cron job this will go fairly slow, but when using multiple worker daemons for the queue, this sends out massive amounts of e-mails.

### Steps to reproduce

1. Don't forget to use Mailhog or Mailtrap while testing this 😅
2. Enable "Sent notifications"
3. Create a form with at least one e-mail notification with all non-empty fields
4. Enter an emoji in a single or multiline text field
5. Submit the form

### Form settings

- Multi-page form: Both
- Submission Method: Both
- Client-side Validation: Yes
- Custom Form Templates: No

### Craft CMS version

3.7.51

### Plugin version

1.6.6

### Multi-site?

Yes

### Additional context

I've tested multiple environments and projects, this only happens on sites running Formie 1.6.6.

**Comment by engram-design** [comment]
I'm not getting any sort of infinite loop behaviour, but there's indeed an issue when the email body contains an emoji. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.6.7](https://github.com/verbb/formie/releases/tag/1.6.7)
